forms/serializers/form.py

```diff
@@ -245,20 +239,77 @@ class Meta:
             ),
         ]
 
+    def entry_generator(self, entries, sections=None, fields=None):
+        if "sections" in entries:
+            yield from self.entry_generator(entries["sections"], sections=[entry for entry in entries["sections"]])
+        if "fields" in entries:
+            yield from self.entry_generator(entries["fields"], sections=sections, fields=[entry for entry in entries["fields"]])
+        if not isinstance(sections, list) and isinstance(fields, list):
```

Comment:
I think this "if" should be the last one, to make this function easier to read. Even better would be to find pattern that allows you to make the "last yield without if:s" Sort of "default case" without any ifs.

Reply:
```
    def entry_generator(self, entries, sections=None, fields=None):
        if "sections" in entries:
            yield from self.entry_generator(
                entries["sections"], sections=[entry for entry in entries["sections"]]
            )
        if "fields" in entries:
            yield from self.entry_generator(
                entries["fields"],
                sections=sections,
                fields=[entry for entry in entries["fields"]],
            )
        if isinstance(entries, list):
            for entry in entries:
                yield from self.entry_generator(entry, sections=sections, fields=fields)
        if isinstance(sections, list):
            for section in sections:
                yield from self.entry_generator(entries[section], sections=section)
        if not isinstance(fields, list):
            return

        for field in fields:
            value_dict = entries[field]
            yield field, sections, value_dict
```